aws.yaml

```diff
@@ -199,6 +199,44 @@
     - name: cluster-operator
       provider: aws
       version: 0.21.4
+- version: 9.0.1
+  active: true
+  date: 2020-03-09T19:00:00Z
+  apps:
+    - app: cert-exporter
+      version: 1.2.1
+    - app: chart-operator
+      version: 0.11.5
+    - app: cluster-autoscaler
+      componentVersion: 1.16.2
+      version: 1.1.3
+    - app: coredns
+      componentVersion: 1.6.5
+      version: 1.1.3
+    - app: kube-state-metrics
+      componentVersion: 1.9.2
+      version: 1.0.2
+    - app: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - app: net-exporter
+      version: 1.6.0
+    - app: nginx-ingress-controller
+      componentVersion: 0.29.0
+      version: 1.5.0
+    - app: node-exporter
+      componentVersion: 0.18.1
+      version: 1.2.0
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 5.5.0
+    - name: cert-operator
+      version: 0.1.0
+    - name: cluster-operator
+      provider: aws
+      version: 0.23.4
```

Comment:
This cluster-operator drops setting replica count, assuming nginx IC with HPA enabled is used, which is only true for v1.6.0 of the app, therefore nginx has to be updated too.

We're preparing release notes in https://github.com/giantswarm/releases/pull/141 for all releases with HPA enabled.

Reply:
Thanks, I updated nginx as 1.6.0.

Reply:
@tomahawk28 I've applied master (to have nginx app 1.6.1 definition), and changed for 9.0.1 aws release:
- cluster-operator from 0.23.4 to 0.23.5
- nginx from 1.6.0 to 1.6.1
- bumped release date to tomorrow

This aligns 9.0.1 with rest of the releases in a batch of releases being prepared with HPA enabled nginx.